libcontainer/integration/exec_test.go

```diff
@@ -1014,3 +1015,238 @@ func TestSTDIOPermissions(t *testing.T) {
 		t.Fatalf("stderr should equal be equal %q %q", actual, "hi")
 	}
 }
+
+func unmountOp(path string) error {
```

Comment:
This func seems redundant, maybe it's easier to just place `syscall.Unmount` everywhere, this will allow reader to not look for func definition